docs/integrations/destinations/azureblobstorage.md

```diff
@@ -137,7 +137,8 @@ They will be like this in the output file:
 
 | Version | Date | Pull Request | Subject |
 | :--- | :--- | :--- | :--- |
-| 0.1.1 | 2021-12-29 | [\#5332](https://github.com/airbytehq/airbyte/pull/9190) | Added BufferedOutputStream wrapper to blob output stream to improve performance and fix issues with 50,000 block limit. Also disabled autoflush on PrintWriter. |
+| 0.1.2 | 2022-01-20 | [\#9682](https://github.com/airbytehq/airbyte/pull/9682) | Each data synchronization for each thread is written to a new blob to the folder with stream name. |
```

Comment:
Nitpick. Does the `thread` here mean `stream`? If that's the case, it's better to avoid `thread` since it is not a common term within the Airbyte doc.

```suggestion
| 0.1.2 | 2022-01-20 | [\#9682](https://github.com/airbytehq/airbyte/pull/9682) | Each data synchronization for each stream is written to a new blob to the folder with stream name. |
```

Reply:
Thanks for the remarks. I corrected the documentation, but did not make changes to add file extensions.